apps/kg-explorer/src/app/components/filter-menu/filter-menu.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { render } from '@testing-library/angular';
+
+import { FilterMenuComponent } from './filter-menu.component';
+
+const mockFilterCategories = [
+  {
+    label: 'Test Category',
+  },
+  {
+    label: 'Test Category',
+  },
+  {
+    label: 'Test Category',
+  },
+  {
+    label: 'Test Category',
+  },
+  {
+    label: 'Test Category',
+  },
+  {
+    label: 'Test Category',
+  },
+];
+
+describe('FilterMenuComponent', () => {
+  it('should render', async () => {
+    const promise = render(FilterMenuComponent, {
```

Comment:
Create a small `setup` function that does the common logic instead of repeating the same logic in each test